<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserSele</class>
 <widget class="QWidget" name="UserSele">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>257</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QRadioButton" name="ManRBtn">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>130</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>管理员</string>
   </property>
  </widget>
  <widget class="QLabel" name="InerLab">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>401</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="CloseBtn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="UserLab">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="NorRBtn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>130</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>普通用户</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SeleBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>190</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>筛选</string>
   </property>
  </widget>
  <widget class="QLabel" name="TitleLab">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>461</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="UserEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>新建用户</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancleBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>190</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="AccountLab">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>账号类型：</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="AllRBtn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>所有</string>
   </property>
  </widget>
  <zorder>InerLab</zorder>
  <zorder>TitleLab</zorder>
  <zorder>ManRBtn</zorder>
  <zorder>CloseBtn</zorder>
  <zorder>UserLab</zorder>
  <zorder>NorRBtn</zorder>
  <zorder>SeleBtn</zorder>
  <zorder>UserEdit</zorder>
  <zorder>label</zorder>
  <zorder>CancleBtn</zorder>
  <zorder>AccountLab</zorder>
  <zorder>AllRBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
